backend/routes/index.js

```diff
@@ -0,0 +1,12 @@
+const auth = require('./auth');
+const user = require('./user');
+//const plan = require('./plan');
```

Comment:
Kommentare würde ich entfernen, wenn die nicht mehr benötigt werden. Erst recht Code, was nicht benötigt wird.

Reply:
Okay, ja die kommen dann in der naechsten Feature Branch rein. sind weg